src/Raven.Server/Documents/Indexes/Static/MapIndexDefinition.cs

```diff
@@ -64,7 +64,7 @@ private static IndexField[] GetFields(IndexDefinition definition, string[] outpu
                 {
                     var name = $"compound({string.Join(",",field)})";
                     var indexField = IndexField.Create(name, new IndexFieldOptions(), allFields, idX++);
-                    indexField.Indexing = FieldIndexing.No; // handled separately
+                    indexField.Indexing = FieldIndexing.Default; // handled separately
```

Comment:
This creates fields definition for compound fields. However, setting as no or default has no final effect since static indexes handle compoundfields indexing separately and during indexing we do not loop over index fields but on map properties. I had to change it since later it causes exception when writing compound field (we have to build a CompactTree/FST as for any other fields).

CompoundFields are not supported for AutoIndexes